lib/pbio/include/pbio/protocol.h

```diff
@@ -147,8 +183,24 @@ typedef enum {
      * @since Pybricks Profile v1.4.0
      */
     PBIO_PYBRICKS_COMMAND_WRITE_APP_DATA = 7,
-} pbio_pybricks_command_t;
 
+    /**
+     * Requests that a previously downloaded user program or builtin user
+     * program is started.
+     *
+     * Parameters:
+     * - payload: Program identifier (one byte). Slots 0--127 are reserved for
+     *            downloaded user programs. Slots 128--255 are for builtin user
+     *            programs.
+     *
+     * Errors:
+     * - ::PBIO_PYBRICKS_ERROR_BUSY if another program is already running.
+     * - ::PBIO_PYBRICKS_ERROR_INVALID_COMMAND if the requested program is not available.
+     *
+     * @since Pybricks Profile v1.4.0.
+     */
+    PBIO_PYBRICKS_COMMAND_START_USER_OR_BUILTIN_PROGRAM = 8,
```

Comment:
:+1: for less verbose.

> And I still think it would be fine to just add an optional parameter to `PBIO_PYBRICKS_COMMAND_START_USER_PROGRAM` instead of adding a new command if we want to go that route instead.

The new one will also return different errors (e.g. when requested id is not available). But I'll look at merging them.